analysis/test/hh/external_library_recipe.sh

```diff
@@ -0,0 +1,29 @@
+cd $CMSSW_BASE/src/nano/
+wget http://www.hep.phy.cam.ac.uk/~lester/dtm662/mt2/Releases/oxbridgekinetics.tar.gz
+tar -xzvf oxbridgekinetics.tar.gz
```

Comment:
if only a few files are needed, just add it to the external lib

Reply:
I don't exactly know what I should use among them for now